src/lib/shopping-fpti/event-handlers.js

```diff
@@ -8,16 +8,41 @@ function findConfigurationAttribute(config : Config, payload : Object = {}, attr
   return shopperConfig[attribName] || payload[attribName];
 }
 
+export const includedAttributes = [
```

Comment:
minor : maybe `allowedAttributes` ?

Reply:
Yeah, that makes more sense. I'll update it.